gridstatus/ercot.py

```diff
@@ -46,6 +46,14 @@
 # https://www.ercot.com/mp/data-products/data-product-details?id=NP6-322-CD
 SCED_SYSTEM_LAMBDA_RTID = 13114
 
+# DAM Shadow Prices
+# https://www.ercot.com/mp/data-products/data-product-details?id=NP4-191-CD
+DAM_SHADOW_PRICES_RTID = 12332
+
+# SCED Shadow Prices
+# https://www.ercot.com/mp/data-products/data-product-details?id=NP6-86-CD
+SCED_SHADOW_PRICES_RTID = 12302
```

Comment:
| SCED Timestamp            | Publish Time              | Constraint ID | Constraint Name  | ContingencyName | Shadow Price | Max Shadow Price | Limit  | Value  | Violated MW | From Station | To Station | From Station kV | To Station kV | CCT Status | 
|---------------------------|---------------------------|---------------|------------------|-----------------|--------------|------------------|--------|--------|-------------|--------------|------------|-----------------|---------------|------------| 
| 2024-02-25 23:00:25-06:00 | 2024-02-26 00:05:02-06:00 | 1             | N_SHARPE_XF1     | SN_SLON5        | 0.0          | 4500             | 402.0  | 335.9  | -66.1       | N_SHARPE     | N_SHARPE   | 345             | 138           | NONCOMP    | 
| 2024-02-25 23:00:25-06:00 | 2024-02-26 00:05:02-06:00 | 4             | LASPUL_RAYMND1_1 | SN_SAJO5        | 0.0          | 3500             | 216.0  | 126.4  | -89.5       | LASPULGA     | RAYMND2    | 138             | 138           | NONCOMP    | 
| 2024-02-25 23:00:25-06:00 | 2024-02-26 00:05:02-06:00 | 14            | HEXT_MASONS1_1   | SFORYEL8        | 0.0          | 2800             | 25.7   | 14.4   | -11.3       | MASONSW      | HEXT       | 69              | 69            | NONCOMP    | 
| 2024-02-25 23:00:25-06:00 | 2024-02-26 00:05:02-06:00 | 2             | LPLWA_LPLDB_1    | SDBMFID5        | 0.0          | 3500             | 217.6  | 209.3  | -8.3        | LPLDB        | LPLWA      | 115             | 115           | NONCOMP    | 
| 2024-02-25 23:00:25-06:00 | 2024-02-26 00:05:02-06:00 | 6             | BURNS_RIOHONDO_1 | SBONNED5        | 0.0          | 3500             | 197.8  | 157.8  | -40.1       | RIOHONDO     | MV_BURNS   | 138             | 138           | NONCOMP    | 
| 2024-02-25 23:00:25-06:00 | 2024-02-26 00:05:02-06:00 | 13            | PEARSALL_69_4    | MPEAMOO8        | 0.0          | 3500             | 140.7  | 118.2  | -22.5       | PEARSALL     | PEARSALL   | 138             | 69            | NONCOMP    | 
| 2024-02-25 23:00:25-06:00 | 2024-02-26 00:05:02-06:00 | 12            | HAINE__LA_PAL1_1 | MHARNED5        | 0.0          | 3500             | 210.7  | 182.0  | -28.7       | LA_PALMA     | HAINE_DR   | 138             | 138           | NONCOMP    | 
| 2024-02-25 23:00:25-06:00 | 2024-02-26 00:05:02-06:00 | 5             | BURNS_RIOHONDO_1 | MHARNED5        | 0.0          | 3500             | 197.6  | 165.6  | -32.0       | RIOHONDO     | MV_BURNS   | 138             | 138           | NONCOMP    | 
| 2024-02-25 23:00:25-06:00 | 2024-02-26 00:05:02-06:00 | 11            | PNHNDL           | BASE CASE       | 0.0          | 5251             | 3239.3 | 2924.2 | -315.1      |              |            | 0               | 0             | NONCOMP    | 
| 2024-02-25 23:00:25-06:00 | 2024-02-26 00:05:02-06:00 | 9             | 6036__A          | DMGSBIT5        | 33.87014     | 4500             | 1278.2 | 1278.2 | 0.0         | TKWSW        | MGSES      | 345             | 345           | COMP       | 
| 2024-02-25 23:00:25-06:00 | 2024-02-26 00:05:02-06:00 | 16            | NELRIO           | BASE CASE       | 0.0          | 5251             | 879.8  | 540.4  | -339.3      |              |            | 0               | 0             | NONCOMP    | 
| 2024-02-25 23:00:25-06:00 | 2024-02-26 00:05:02-06:00 | 15            | NE_LOB           | BASE CASE       | 0.0          | 5251             | 1245.3 | 1222.9 | -22.3       |              |            | 0               | 0             | NONCOMP    |

Reply:
| Command | Description |
| --- | --- |
| git status | List all new or modified files |
| git diff | Show file differences that haven't been staged |